ui/src/app/settings/components/project-role-jwt-tokens/project-role-jwt-tokens.tsx

```diff
@@ -58,6 +58,20 @@ export const ProjectRoleJWTTokens = (props: ProjectRoleJWTTokensProps) => {
                         })}>
                         {api => (
                             <form onSubmit={api.submitForm} role='form' className='width-control'>
+                                <div className='argo-table-list'>
```

Comment:
I've noticed several issues with UI:

* Token ID and Expires fields are separated visually. So it looks like fields belong to two different forms:

![image](https://user-images.githubusercontent.com/426437/79012695-bdfc8d80-7b1b-11ea-855c-1a4643d63e89.png)

* The following message is shown if user makes not changes in Token ID field and clicks Create: 'Are you sure you want to create a JWT token that name with **'undefined'** and has no expiration for role 'test' in project 'default'?' I think we should make Token ID required field. 

* After creating new token Token ID input field is reset and has previously entered Token ID. This is ok for Expires In but not for Token ID field because user definitely needs to change it.

Token creation form is implemented using table CSS classes. It makes it difficult to have more than 1 field. I suggest use standard form CSS classes ( see [application-create-panel.tsx](https://github.com/argoproj/argo-cd/blob/master/ui/src/app/applications/components/application-create-panel/application-create-panel.tsx#L169)  ) 

I would move UI changes to separate PR.

Reply:
Addressed the above issues, the UI with re-design:

<img width="1219" alt="Screen Shot 2020-04-13 at 4 25 51 AM" src="https://user-images.githubusercontent.com/62673041/79161483-6a46aa00-7d90-11ea-8579-5752b9817ac6.png">